site/en/fugu-showcase/index.njk

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Project Fugu API Showcase'
+subhead: >
+  The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in the context of Project Fugu.
+description: >
+  The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in the context of Project Fugu.
+hero: image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/wu6CeuL1HpHtc8b4WIIm.png
+alt: Blowfish swarm swimming in the ocean.
+---
+
+{% extends "layouts/base.njk" %}
+{% from 'macros/cards/hero-card.njk' import heroCard with context %}
+{% from 'macros/icon.njk' import icon with context %}
+
+{% block content %}
+
+<div class="fugu-showcase">
+  <div class="width-full pad-400 fugu-showcase__hero">
+    {% if hero %}
+      {{ heroCard(
+        "Project Fugu API Showcase",
+        "The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in the context of Project Fugu.",
+        "Submit a new app",
+        "https://docs.google.com/forms/d/e/1FAIpQLScNd1rClbmFWh6FcMmjUNrwg9RLz8Jk4BkHz_-EOpmkVd_-9g/viewform",
+        hero,
+        "Project Fugu API Showcase",
+        "cyan"
+      ) }}
+    {% endif %}
+  </div>
+
+  <div class="width-full pad-left-400 pad-right-400">
+    <div class="md:display-flex search-box">
+      <div class="search-box__inner" role="presentation">
+        <span class="search-box__btn">
+          {{ icon('search', {hidden: false}) }}
+        </span>
+        <input
+          id="search-fugu-apps"
+          class="search-box__input"
+          placeholder="Search apps"
+          aria-label="Search apps"
+          aria-autocomplete="list"
```

Comment:
Since you are `| i18n` some of the UI copy in L46ff I think it's either none or all 😄